webClient/src/app/core/menu-bar/menu-bar.component.ts

```diff
@@ -107,15 +133,30 @@ export class MenuBarComponent implements OnInit {
     // this.editorControl.saveAllFile.subscribe(x => {
     //   this.saveAll();
     // });
+
   }
 
-  private resetLanguageSelectionMenu() {
-    const monaco = this.editorControl.editorCore.getValue();
-    if (!monaco) {
-      this.log.warn(`Language registered without monaco present`);
-      return;
+  public getMenuItemStyle(menuItem) {
```

Comment:
Expanding way on dynamically changing menu options will mean good things for future enhancements to Editor